src/analyses/goto_rw.h

```diff
@@ -165,6 +165,7 @@ class rw_range_sett
 
 protected:
   const namespacet &ns;
+  guard_managert &guard_manager;
```

Comment:
Is it both being passed around and owned?

Reply:
I removed the unnecessary passing around